src/lib/crypto/kimchi_backend/pasta/pallas_based_plonk.ml

```diff
@@ -111,17 +111,19 @@ module Proof = Plonk_dlog_proof.Make (struct
             done ;
             witness )
       in
-      create pk.index witness_cols runtime_tables prev_chals prev_comms
+      backend_create pk.index witness_cols runtime_tables prev_chals prev_comms
 
     let create_async (pk : Keypair.t) ~primary ~auxiliary ~prev_chals
         ~prev_comms =
       create_aux pk ~primary ~auxiliary ~prev_chals ~prev_comms
-        ~f:(fun pk auxiliary_input runtime_tables prev_challenges prev_sgs ->
+        ~f:(fun index witness runtime_tables prev_chals prev_sgs ->
```

Comment:
I imagine the prev_chals change is related to homogeneity, right?

Reply:
Yes, it's just more homogenous w.r.t. the other call in the same file.